File: semanticdb/integration/src/main/scala/example/MetacJava.scala
package example

import com.javacp.InnerClasses

class MetacJava {
  InnerClasses.StaticInner.isStatic()
  new InnerClasses.StaticInner().isNotStatic()
  val inner = new InnerClasses()
  val overload1 = new inner.Overload1()
  val overload2 = new inner.Overload2()
  inner.overload(new overload1.A())
  inner.overload(new overload2.A())
  val staticInner = new InnerClasses.StaticInner()
  val nonStatic = new staticInner.NonStatic()
  nonStatic.method(nonStatic)
}

File: semanticdb/integration/src/main/java/com/javacp/MetacJava.java
package com.javacp;

public class MetacJava {
    public static class StaticInner {
        public void isNotStatic() {}
        public static void isStatic() {}
        public class NonStatic {
            public void method(NonStatic e) {}
        }
    }
    public class Overload1 { public class A {} }
    public class Overload2 { public class A {} }
    public void overload(Overload1.A a) {}
    public void overload(Overload2.A a) {}
}
